.github/workflows/release.yml

```diff
@@ -18,6 +18,10 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.16
+      - name: Set up cosign
```

Comment:
This is missing 2 things:

```
    permissions:
      id-token: write  # undocumented OIDC support.
      contents: write
```

id-token provides the OIDC token so cosign can sign stuff
contents allows goreleaser to push properly to the releases. This wasn't needed before, not sure why its needed now but certainly shows in the docs: https://goreleaser.com/ci/actions/#token-permissions

Reply:
wait, it means it requires a PAT token for it? do we have already one for it?

Reply:
no no, it doesnt seem to, it just requires to set that and it magically works...At least during my changes to luet-mtree it failed to upload the release without it...but it did before....

https://github.com/rancher-sandbox/luet-mtree/runs/5968272129?check_suite_focus=true

```
 ⨯ release failed after 122.62s error=scm releases: failed to publish artifacts: POST https://api.github.com/repos/rancher-sandbox/luet-mtree/releases: 403 Resource not accessible by integration []
```

Reply:
the default for content in actions should be read/write but that cna be changed https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token maybe it was....or someone was playing with it, we can try without it, see if it works and if doesnt just push to master, repush tag to the new commit, call it a day.

Reply:
got it now, thanks! updated the PR